File: Python/Program/TokoElektronik.py
from Produk import Produk

class TokoElektronik(Produk):
    def __init__(self):
        super().__init__()
        self._id_toko = 0
        self._nama_toko = ""
        self._pemilik = ""
        self._alamat = ""
        self._no_izin_usaha = ""
        self._stok = 0

    def get_id_toko(self): return self._id_toko
    def set_id_toko(self, v): self._id_toko = v

    def get_nama_toko(self): return self._nama_toko
    def set_nama_toko(self, v): self._nama_toko = v

    def get_pemilik(self): return self._pemilik
    def set_pemilik(self, v): self._pemilik = v

    def get_alamat(self): return self._alamat
    def set_alamat(self, v): self._alamat = v

    def get_no_izin_usaha(self): return self._no_izin_usaha
    def set_no_izin_usaha(self, v): self._no_izin_usaha = v

    def get_stok(self): return self._stok
    def set_stok(self, v): self._stok = v

File: Php/Program/TokoElektronik.php
<?php
require_once "Produk.php";

class TokoElektronik extends Produk {
    private int $idToko;
    private string $namaToko,$pemilik,$alamat;

    public function __construct(int $idToko,string $namaToko,string $pemilik,string $alamat,
                                string $namaProduk,string $brand,string $kode,float $harga,string $deskripsi,string $foto){
        parent::__construct($namaProduk,$brand,$kode,$harga,$deskripsi,$foto);
        $this->idToko=$idToko; $this->namaToko=$namaToko; $this->pemilik=$pemilik; $this->alamat=$alamat;
    }

    public function getIdToko(){return $this->idToko;}
    public function setIdToko($v){$this->idToko=$v;}
    public function getNamaToko(){return $this->namaToko;}
    public function setNamaToko($v){$this->namaToko=$v;}
    public function getPemilik(){return $this->pemilik;}
    public function setPemilik($v){$this->pemilik=$v;}
    public function getAlamat(){return $this->alamat;}
    public function setAlamat($v){$this->alamat=$v;}
}
